Sources/Datadog/Tracer.swift

```diff
@@ -139,14 +139,22 @@ public class Tracer: OTTracer {
         return nil
     }
 
+    public var activeSpan: OTSpan? {
```

Comment:
This property should be also added to `OTTracer` interface, otherwise our users won't see it.

Our users do not use `Tracer` directly. When they register global tracer:
```swift
Global.sharedTracer = Tracer.initialize(configuration: .init())
```
what is available to them through `Global.sharedTracer` is `OTTracer` interface.

Reply:
I have added it already, not sure if this is the proper way, though